package.json

```diff
@@ -5,7 +5,10 @@
   "scripts": {
     "kenson": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "prepare": "husky install",
+    "type-check": "tsc --project tsconfig.json --pretty --noEmit",
+    "lint": "eslint --ext js,jsx,ts,tsx --fix"
```

Comment:
I thought next.js already comes with eslint support out of the box, why are we configuring eslint ourselves?

Reply:
You're right, I just found out. I followed a tutorial to set this up. I'll make the changes and remove this.